site/src/atomics/typography.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Typography Atomics
+description: Typography related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Typography Atomics
+
+The typography scale is designed to for great readability across the platform. This page covers the various classes available to modify text.
+
+## Font size
+
+There are two avenues for modifying font sizes in Atlas.
+
+#### Heading sizes
+
+The following classes can be used to accurately adopt a heading's size (h1- h6).
+
+```html
+<p class="font-size-h1">Heading 1</p>
+<p class="font-size-h2">Heading 2</p>
+<p class="font-size-h3">Heading 3</p>
+<p class="font-size-h4">Heading 4</p>
+<p class="font-size-h5">Heading 5</p>
+<p class="font-size-h6">Heading 6</p>
+```
+
+#### Non-heading sizes
+
+The following classes targets text that are not headings.
+
+```html
+<p class="font-size-xl">Extra large text</p>
+<p class="font-size-l">Large text</p>
+<p class="font-size-m">Medium text</p>
+<p class="font-size-s">Small text</p>
+<p class="font-size-xs">Extra small text</p>
+```
+
+## Font weight and italics
+
+There are 5 classes available to transform the font weight. There is 1 class to transform the font style.
+
+```html
+<p class="font-weight-semilight">Semilight weight text</p>
+<p class="font-weight-light">Light weight text</p>
+<p class="font-weight-normal">Normal weight text</p>
+<p class="font-weight-semibold">Semibold weight text</p>
+<p class="font-weight-bold">Bold weight text</p>
+<p class="font-style-italic">Italic text</p>
+```
+
+## Font family
+
+The font family can be transformed with the following classes.
+
+```html
+<p class="font-family-monospace">Font Family Monospace</p>
+<p class="font-family-quote">Font Family for block quotes</p>
```

Comment:
I think we can remove this one, this is used for the internals of a very particular component. It's probably doesn't need to be an atomic either, but that is not a critical issue.

Reply:
Removed